src/tmp/SampleEnginePlugin.ts

```diff
@@ -121,7 +121,7 @@ export class SampleEngine1 extends EngineApi.Engine {
 				}
 				violations.push({
 					ruleName,
-					message: 'Fake Rule Fakily Violated',
+					message: 'Fake Rule\nFakily Violated',
```

Comment:
One engine's violations are now multi-line for ease of testing how multi-line violations are handled.